code/modules/surgery/organs/liver.dm

```diff
@@ -29,11 +29,13 @@
 				//handle liver toxin filtration
 				var/toxamount
 				var/static/list/listOfToxinsInThisBitch = typesof(/datum/reagent/toxin)
-				for(var/datum/reagent/toxin/toxin in listOfToxinsInThisBitch)
+				for(var/I in listOfToxinsInThisBitch)
+					var/datum/reagent/toxin = I

```

Comment:
this should be var/datum/reagent/toxin/toxin

Reply:
it doesn't need to be, /datum/reagent has the id var